shopelectro/settings/base.py

```diff
@@ -542,3 +542,13 @@ def get_robots_content():
             CeleryIntegration(),
         ]
     )
+
+# At 17.05.2019 it's correct pack group data
+PACK_GROUP_UUID = os.environ.get(
```

Comment:
let's use this option without env var. This option is specific for app container only.
We have `local.py` to change it fastly on any env

```suggestion
PACK_GROUP_UUID = 'ae30f766-0bb8-11e6-80ea-02d2cc20e118'
```